webapp/src/components/Analysis/UtterancesTable.tsx

```
@@ -390,6 +399,37 @@ const UtterancesTable: React.FC<Props> = ({
   ];
 
   const searchString = constructSearchString(pipeline);
+
+  const importProposedActions = (file: File) => {
+    const fileReader = new FileReader();
+    fileReader.onload = ({ target }) => {
```

Bug: When importing a CSV file of proposed actions, importing the same file path twice fails silently. The file browser closes normally but the `fileReader.onload` is never called.
Use case: One might have edited the file and wants to re-import it.
Workaround: Refresh the app.
Let's fix that in another PR. I filed a bug: #479

---

Ohh, Let me check that use case. Thanks for finding that out and creating a follow-up issue for it.